src/legacy/js/functions/_setupFlorence.js

```diff
@@ -371,5 +371,15 @@ function setupFlorence() {
             this.collection = collectionTemp
         }
     }
+
+    $(document).on('blur', 'input, textarea', function() {
+        if (this.type === "file") {
```

Comment:
This if statement doesn't cover a select, checkboxes and other input types that have return a user input value so don't need to run the same functions.

Reply:
Ignore me, I've found out this was only done on file inputs because it causes a runtime error. I'll add a comment to the code so we know in the future though.